### PR TITLE
fixes issue 2280

### PR DESCRIPTION
**Pull request by nesalang**
fixes issue #2280

### CODE REVIEW

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/TextInputRenderer.java [hunk 1]
```diff
@@ -187,76 +187,68 @@ public void afterTextChanged(Editable s)
...
-                        inlineButton = new Button(themedContext, null, 0);
-                    }
-                    else
-                    {
-                        inlineButton = new Button(context);
-                        inlineButton.setBackgroundColor(Color.TRANSPARENT);
-                        inlineButton.setTextColor(Color.BLACK);
-                        inlineButton.setPadding(16, 0, 0, 8);
+        if(baseInputElement instanceof TextInput)
+        {
```
Comment (paulcam206) [inline source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/TextInputRenderer.java]: should `{` go on this line, or at the end of the line above? inconsistent

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/TextInputRenderer.java [hunk 2]
```diff
@@ -187,76 +187,68 @@ public void afterTextChanged(Editable s)
...
-                        Context themedContext = new ContextThemeWrapper(context, R.style.adaptiveInlineAction);
-                        inlineButton = new Button(themedContext, null, 0);
-                    }
-                    else
-                    {
-                        inlineButton = new Button(context);
-                        inlineButton.setBackgroundColor(Color.TRANSPARENT);
-                        inlineButton.setTextColor(Color.BLACK);
-                        inlineButton.setPadding(16, 0, 0, 8);
+        if(baseInputElement instanceof TextInput)
```
Comment (paulcam206) [inline source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/TextInputRenderer.java]: what should we do if `baseInputElement` is *not* an instance of `TextInput`? is that an expected case?

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/TextInputRenderer.java (hunk 1 above)
Comment (nesalang) [reply source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/TextInputRenderer.java]: thanks

Comment (nesalang) [reply source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/TextInputRenderer.java]: thanks

---
In reply to: [246190275](https://github.com/Microsoft/AdaptiveCards/pull/2281#discussion_r246190275) [](ancestors = 246190275)

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/TextInputRenderer.java (hunk 2 above)
Comment (nesalang) [reply source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/TextInputRenderer.java]: yes, this expected case. NumberInput instance can call this method.


---
In reply to: [246190835](https://github.com/Microsoft/AdaptiveCards/pull/2281#discussion_r246190835) [](ancestors = 246190835)

**Pull request merged**